torch/csrc/profiler/util.cpp

```diff
@@ -312,13 +312,13 @@ std::string ivalueToStr(const c10::IValue& val, bool isString) {
     // json only takes "true" and "false" so we convert the string to lower case
     if (val.isBool()) {
       for (char& c : mystr) {
-        c = std::tolower(c);
+        c = static_cast<char>(std::tolower(c));
```

Comment:
```suggestion
        c = std::tolower(c, std::locale());
```
from `<locale>` is the newer recommended overload of this type in modern C++

Reply:
it‘s dealing with ASCII... locale is too complicated that I usually don't touch..